billing/faqs.md

```diff
@@ -59,4 +59,34 @@ A coupon can be used when you:
 
 You are asked to enter your coupon code when you confirm or enter your payment method. 
 
-If you use a coupon to pay for a subscription, when the coupon expires, your payment method is charged the full cost of your subscription. If you do not have a saved payment method, your account is downgraded to a free subscription. 
+If you use a coupon to pay for a subscription, when the coupon expires, your payment method is charged the full cost of your subscription. If you do not have a saved payment method, your account is downgraded to a free subscription.
+
+### How do I include my VAT number on my invoice?
+
+Organization owners can update invoice details to include a VAT number. To update the billing information for your organization:
```

Comment:
**suggestion:** VAT number can be added to both user and organization account as long as a billing address is in EU zone (see `eu_vat` here https://stripe.com/docs/billing/customer/tax-ids).

```suggestion
User account or organization owners can update invoice details to include a VAT number. To update the billing information for your organization:
```

Reply:
Good to know, will update.